### PR TITLE
Solution Documentation

### PR DESCRIPTION
**Pull request by tznind**
This is documentation for the SmiServices repo that describes in detail what our repo does as requested by Susan.

- [x] 1 Introduction
- [x] 2 Solutions
- [x] 3 Further Considerations
- [x] 4 Future Work
- [x] 5 End Matter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,7 +225,107 @@ Configure 'default.yaml'
...
+SmiServices improves upon this approach by loading an imaging archive to a single anonymous indexed table per modality with aggregate
+tables containing Study and Series level information.  This format is the fastest queryable representation for large scale linkage and extraction.
+
+### 5.2 Acknowledgments
+
+Contributors to the SmiServices repository can be seen in the [GitHub Contributors page](https://github.com/SMI/SmiServices/graphs/contributors).
+
+Contributors to the [upstream dependencies](./PACKAGES.md) can be seen in the corresponding section of those repositorys.
+
+This work has been funded under the PICTURES program.  A 5-year, £3.8M programme of work funded by the Medical Research Council (MRC) with additional support from the Engineering and Physical Sciences Research Council (EPSRC) as part of Health Data Research UK (HDR UK).
```
Comment (rkm) [inline README.md:328]: program -> programme, and we should also include detail of the initial Farr grant for the MVP.

**Review by rkm — CHANGES_REQUESTED** [review]
Great work! Just a couple comments to address before we merge.

As a separate issue, we should consider a central ReadTheDocs type site where we can host all relevant docs and images across the different repos.

File: README.md [hunk 2]
```diff
@@ -12,28 +12,203 @@ Version: `4.0.0`
...
+The following processes are solved by the suite:
+
+- Robust parallel loading of dicom metadata into a relational database where it can be linked to EHR data
+- Anonymisation of the dicom metadata within the relational database
+- Cohort building by linking the relational database tables with other EHR data (e.g. biochemistry results, prescriptions etc)
+- Producing anonymous dicom image files for a subset of the repository
+
+Stakeholders likely to interact with this suite include Research Coordinators (building research extracts) and Data Analysts (loading data, verifying anonymisation etc)
+
+The solution is built on top of the [Research Data Management Platform](https://github.com/HicServices/RDMP) which drives many of its underlying processes.
```
[the remark this answers is not in the document]
Comment (tznind) [reply README.md]: I have removed this sentence.  I agree it doesn't really fit into the overview and its role in ETL is covered later on anyway.

File: README.md (hunk 1 above)
Comment (tznind) [reply README.md]: I've asked Emily to provide a sentence or two about this.

Comment (tznind) [reply README.md]: Added in 4635577

**Review by rkm — APPROVED** [review]

**Pull request merged**